### PR TITLE
Securing tmp Folder of Webserver

### PR DESCRIPTION
**Pull request by lenusch**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Daniel McDonald - mail@danielmcdonald.de

### CODE REVIEW

File: tutorials/secure-tmp-folder/01.en.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [submitter's name and email address here]
```
Comment (n-se) [inline tutorials/secure-tmp-folder/01.en.md]: Please add your name and email address here, too

Comment (lenusch) [reply tutorials/secure-tmp-folder/01.en.md]: Done.

**Comment by lenusch** [comment]
Sorry, all tasks done.

File: tutorials/secure-tmp-folder/01.en.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+date: "2019-03-22"
+title: "Secure TMP Folder on Webserver against execution :+1:"
+short_description: "This Tutorial shows you, how to secure the tmp Folder within a Webserver, against script execution"
+tags: ["secure webserver", "security", "webserver"]
+author: "Daniel McDonald"
+author_link: "https://daka-media.de"
+author_img: ""
+author_description: "Daniel is a web developer and designer. His Company was founded in 2013"
+language: "en"
+available_languages: ""
```
Comment (n-se) [inline tutorials/secure-tmp-folder/01.en.md]:
```suggestion
available_languages: ["en"]
```

**Comment by lenusch** [comment]
["If you have translated the tutorial into other languages, please enter the languages according to ISO 639-1 here."] this can be missunderstood. "other" language is not there, therefore nothing was inserted. 
But anyways. I case of confusion, you are right!

**Comment by n-se** [comment]
You are kinda right, can you open an issue for that or fix it with a pull request?

**Comment by fdrechsler** [comment]
Please add the correct language-type to your Code Blocks so they become highlighted.

For example:

```
     ```console
     foo@bar:~$ whoami
     foo
     ```
```

 ```console
foo@bar:~$ whoami
foo
```

**Comment by lenusch** [comment]
> Please add the correct language-type to your Code Blocks so they become highlighted.


Thank you, done!

File: tutorials/secure-tmp-folder/01.en.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+Be sure you have root permissions. 
+
+
+### Step 2 - Check if a tmp folder already exists!
+
+With following Command you can see if there's already a tmp folder:
+
+```console
+root@server:~$ mount | egrep --color -w '^(tmpfs|/tmp)|/tmp'
```
[the remark this answers is not in the document]
Comment (lenusch) [reply tutorials/secure-tmp-folder/01.en.md]: Mama mia! :-) Of course, done.

File: tutorials/secure-tmp-folder/01.en.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+author_img: ""
+author_description: "Daniel is a web developer and designer. His Company was founded in 2013"
+language: "en"
+available_languages: ["en"]
+header_img: ""
+---
+
+<!-- This where the actual tutorial begins, with the title: -->
+
+# Securing tmp Folder on Webserver on (for example Debian 9)
```
Comment (n-se) [inline tutorials/secure-tmp-folder/01.en.md]: Last one, please drop this line. The title from the headings above will be shown on the website.

Comment (lenusch) [reply tutorials/secure-tmp-folder/01.en.md]: OK, done! :-)

**Pull request merged**